test/fminbnd.jl

```diff
@@ -8,6 +8,12 @@ function fminbnd_tests()
     @test norm(output.solution .- [0.5; 0.25]) < 1e-6
     @test output.dual_feas < 1e-6
 
+    kwargs = (x0 = [-3.0; 1.0],)
+
+    output = fminbnd(f, x₀, ℓ, u; solver = :ipopt, kwargs...)
```

Comment:
Don't create a `kwargs`, manually pass the argument.
Notice that `x0` is not a valid argument for IPOPT, try
```julia
output = fminbnd(f, x₀, ℓ, u; solver = :ipopt, nlp_scaling_method="none")
```

Reply:
It isn't? I looked in [NLPModelsIpopt](https://github.com/JuliaSmoothOptimizers/NLPModelsIpopt.jl/blob/master/src/NLPModelsIpopt.jl)

Reply:
Oh I get it now.